diff --git a/src/game/behaviors/mushroom_1up.inc.c b/src/game/behaviors/mushroom_1up.inc.c
index 91ad658..486e0a4 100644
--- a/src/game/behaviors/mushroom_1up.inc.c
+++ b/src/game/behaviors/mushroom_1up.inc.c
@@ -5,7 +5,7 @@ void bhv_1up_interact(void) {
 
     if (obj_check_if_collided_with_object(o, gMarioObject) == TRUE) {
         play_sound(SOUND_GENERAL_COLLECT_1UP, gGlobalSoundSource);
-        gMarioState->numLives++;
+        //gMarioState->numLives++;
         o->activeFlags = ACTIVE_FLAG_DEACTIVATED;
 #if ENABLE_RUMBLE
         queue_rumble_data(5, 80);
diff --git a/src/game/ingame_menu.c b/src/game/ingame_menu.c
index 0ac715b..cfd71e3 100644
--- a/src/game/ingame_menu.c
+++ b/src/game/ingame_menu.c
@@ -3142,7 +3142,7 @@ void print_hud_course_complete_coins(s16 x, s16 y) {
 
             if (gCourseCompleteCoins == 50 || gCourseCompleteCoins == 100 || gCourseCompleteCoins == 150) {
                 play_sound(SOUND_GENERAL_COLLECT_1UP, gGlobalSoundSource);
-                gMarioState->numLives++;
+                //gMarioState->numLives++;
             }
         }
 
diff --git a/src/game/level_update.c b/src/game/level_update.c
index a6186c3..c16d3db 100644
--- a/src/game/level_update.c
+++ b/src/game/level_update.c
@@ -729,9 +729,9 @@ s16 level_trigger_warp(struct MarioState *m, s32 warpOp) {
                 break;
 
             case WARP_OP_DEATH:
-                if (m->numLives == 0) {
-                    sDelayedWarpOp = WARP_OP_GAME_OVER;
-                }
+                // if (m->numLives == 0) {
+                //     sDelayedWarpOp = WARP_OP_GAME_OVER;
+                // }
                 sDelayedWarpTimer = 48;
                 sSourceWarpNodeId = WARP_NODE_DEATH;
                 play_transition(WARP_TRANSITION_FADE_INTO_BOWSER, 0x30, 0x00, 0x00, 0x00);
@@ -741,11 +741,11 @@ s16 level_trigger_warp(struct MarioState *m, s32 warpOp) {
             case WARP_OP_WARP_FLOOR:
                 sSourceWarpNodeId = WARP_NODE_WARP_FLOOR;
                 if (area_get_warp_node(sSourceWarpNodeId) == NULL) {
-                    if (m->numLives == 0) {
-                        sDelayedWarpOp = WARP_OP_GAME_OVER;
-                    } else {
+                    // if (m->numLives == 0) {
+                    //     sDelayedWarpOp = WARP_OP_GAME_OVER;
+                    // } else {
                         sSourceWarpNodeId = WARP_NODE_DEATH;
-                    }
+                    // }
                 }
                 sDelayedWarpTimer = 20;
                 play_transition(WARP_TRANSITION_FADE_INTO_CIRCLE, 0x14, 0x00, 0x00, 0x00);
diff --git a/src/game/mario.c b/src/game/mario.c
index 32c63c9..13a37b9 100644
--- a/src/game/mario.c
+++ b/src/game/mario.c
@@ -1883,7 +1883,7 @@ void init_mario_from_save_file(void) {
         save_file_get_total_star_count(gCurrSaveFileNum - 1, COURSE_MIN - 1, COURSE_MAX - 1);
     gMarioState->numKeys = 0;
 
-    gMarioState->numLives = 4;
+    gMarioState->numLives = 100;
     gMarioState->health = 0x880;
 
     gMarioState->prevNumStarsForDialog = gMarioState->numStars;
diff --git a/src/game/mario_actions_cutscene.c b/src/game/mario_actions_cutscene.c
index fcee2c2..980c07f 100644
--- a/src/game/mario_actions_cutscene.c
+++ b/src/game/mario_actions_cutscene.c
@@ -1164,7 +1164,7 @@ s32 act_death_exit(struct MarioState *m) {
 #if ENABLE_RUMBLE
         queue_rumble_data(5, 80);
 #endif
-        m->numLives--;
+        //m->numLives--;
         // restore 7.75 units of health
         m->healCounter = 31;
     }
@@ -1180,7 +1180,7 @@ s32 act_unused_death_exit(struct MarioState *m) {
 #else
         play_sound(SOUND_MARIO_OOOF2, m->marioObj->header.gfx.cameraToObject);
 #endif
-        m->numLives--;
+        //m->numLives--;
         // restore 7.75 units of health
         m->healCounter = 31;
     }
@@ -1199,7 +1199,7 @@ s32 act_falling_death_exit(struct MarioState *m) {
 #if ENABLE_RUMBLE
         queue_rumble_data(5, 80);
 #endif
-        m->numLives--;
+        //m->numLives--;
         // restore 7.75 units of health
         m->healCounter = 31;
     }
@@ -1246,7 +1246,7 @@ s32 act_special_death_exit(struct MarioState *m) {
 #if ENABLE_RUMBLE
         queue_rumble_data(5, 80);
 #endif
-        m->numLives--;
+        //m->numLives--;
         m->healCounter = 31;
     }
     // show Mario
